src/components/pill/index.js

```diff
@@ -0,0 +1,146 @@
+import { html, css, LitElement, unsafeCSS } from 'lit';
+import { createRef, ref } from 'lit/directives/ref.js';
+import styles from './styles.scss';
+import { classMap } from 'lit/directives/class-map.js';
+import { TYPE } from './enums';
+
+export class VlPill extends LitElement {
+  static get EVENTS() {
+    return {
+      close: 'close',
+      check: 'check',
+    };
+  }
+
+  static get styles() {
+    return [
+      css`
+        ${unsafeCSS(styles)}
+      `,
+    ];
+  }
+
+  static get properties() {
+    return {
+      classes: {},
+      disabled: {
+        type: Boolean,
+        attribute: 'data-vl-disabled',
+        reflect: true,
+      },
+      type: {
+        type: String,
+        attribute: 'data-vl-type',
+        reflect: true,
+      },
+      closable: {
+        type: Boolean,
+        attribute: 'data-vl-closable',
+        reflect: true,
+      },
+      checkable: {
+        type: Boolean,
+        attribute: 'data-vl-checkable',
+        reflect: true,
+      },
+      checked: {
+        type: Boolean,
+        attribute: 'data-vl-checked',
+        reflect: true,
+      },
+    };
+  }
+
+  constructor() {
+    super();
+    this.type = '';
```

Comment:
Hoeft type default een lege string te zijn, of is er default gewoon geen type?

Reply:
Fixed.